src/burnchains/burnchain.rs

```diff
@@ -43,6 +43,9 @@ use burnchains::{
 };
 
 use burnchains::Error as burnchain_error;
+use burnchains::db::{
+    BurnchainDb
```

Comment:
Minor nitpick -- can we be consistent with `DB` vs `Db`?  Shouldn't this be `DB` since that's what we've been using elsewhere?

Reply:
Yep, works for me, will update

Reply:
Updated in 99ed20971